Windows release build fails: zts template parsing error

## Description

The Windows release build fails with a zts template parsing error when compiling the CLI. The error indicates that the template section `.header` cannot be found.

## Error

```
.zig-cache\p\zts-0.14.1-...\src\zts.zig:91:5: error: Data does not contain any section labelled 'header'
    @compileError(directiveNotFound);
```

## Root Cause

The CLI imports `generator.zig` which uses zts templates at comptime:
```zig
const build_zig_tmpl = @embedFile("templates/build_zig.txt");
```

The template files use Unix line endings (LF), but zts may have issues parsing them on Windows at comptime.

## Affected

- `.github/workflows/release.yml` - Windows build job
- v0.27.0 release is missing Windows binary

## Possible Solutions

1. **Fix line ending handling in zts** - Ensure zts handles both LF and CRLF line endings
2. **Convert templates to CRLF for Windows** - Pre-process template files during build
3. **Lazy-load templates at runtime** - Don't use `@embedFile` at comptime, load templates at runtime instead
4. **Separate CLI from generator** - The upgrade command doesn't need generator, only init/generate do

## Related

- v0.27.0 release workflow failed: https://github.com/labelle-toolkit/labelle-engine/actions/runs/20598715822

--- a/.gitattributes
+++ b/.gitattributes
@@ -2,17 +2,27 @@
 * text=auto eol=lf
 
 # Zig source files
-*.zig text eol=lf
+*.zig text
 
 # Template files (critical for zts parsing)
-*.txt text eol=lf
+*.txt text
 
 # Build files
-*.zon text eol=lf
+*.zon text
 
 # Shell scripts
-*.sh text eol=lf
+*.sh text
 
 # Windows batch files need CRLF
 *.bat text eol=crlf
 *.cmd text eol=crlf
+
+# Binary files (prevent line ending conversion)
+*.png binary
+*.jpg binary
+*.jpeg binary
+*.gif binary
+*.ico binary
+*.ttf binary
+*.woff binary
+*.woff2 binary

